packages/lavalink/lib/src/client.dart

```diff
@@ -155,19 +160,26 @@ class LavalinkClient extends HttpLavalinkClient {
   LavalinkConnection get connection => _connection;
   late final LavalinkConnection _connection;
 
+  /// A list of plugins used by this client.
+  List<LavalinkExternalPlugin> get plugins => List.unmodifiable(_plugins);
```

Comment:
```suggestion
  List<LavalinkExternalPlugin> get plugins => UnmodifiableListView(_plugins);
```

Avoid making copies every time.